backend/src/Constant/StoreOwner/StoreProfileConstant.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Constant\StoreOwner\StoreProfileConstant;
+
+abstract class StoreProfileConstant
```

Comment:
Please, update class from abstract to final, and variables from static to const.